core/client_test.go

```diff
@@ -0,0 +1,52 @@
+package core
```

Comment:
It's a bit weird that this shows up as a new file. I think it would be best-practice to move the file, then rename the package. I'd expect git would be smart enough to understand the file was moved / renamed and not created from scratch. It's unnecessary to change anything here but it is a bit odd that we lose the files history just by moving them around.

Reply:
I agree and should be resolved by https://github.com/celestiaorg/celestia-node/pull/55, as it will mark tests as Rene's contribution